tools/ci-scripts/codegen-diff/diff_lib.py

```diff
@@ -61,12 +61,8 @@ def generate_and_commit_generated_code(revision_sha, targets=None):
         if target in targets:
             get_cmd_output(f"mv {target}/build/smithyprojections/{target} {OUTPUT_PATH}/")
             if target == target_codegen_server:
-                get_cmd_output(
-                    f"mv {PYTHON_EXAMPLES_PATH}/pokemon-service-server-sdk/ {OUTPUT_PATH}/codegen-server-test-python/",
-                    check=False)
-                get_cmd_output(
-                    f"mv codegen-server-test/typescript/build/smithyprojections/codegen-server-test-typescript {OUTPUT_PATH}/",
-                    check=False)
+                get_cmd_output(f"mv {target}/python/build/smithyprojections/{target}-python {OUTPUT_PATH}/")
+                get_cmd_output(f"mv {target}/typescript/build/smithyprojections/{target}-typescript {OUTPUT_PATH}/")
```

Comment:
should this be `target == target_codegen_typescript/python` instead?

Reply:
At the end it does the same.. I have some other rearchitectures of the diff lib in the next PR. I'll refactor this part in it as well if that's ok?